source/language/classical.rst

```diff
@@ -110,6 +110,40 @@ multiplication, and division and the corresponding assignment operators.
    angle[10] c;
    c = angle(a + b); // cast to angle[10]
 
+
+Evaluation order
+~~~~~~~~~~~~~~~~
+
+OpenQASM evaluates expressions from left to right.
+The operator precedence is based on C.
```

Comment:
```suggestion
```
I say remove the line about C. We don't want to be held to that. The order is defined by the table below (and the accompanying ANTLR .g4 file)